public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js

```diff
@@ -415,6 +415,20 @@ function Onboarding(svl, audioEffect, compass, form, handAnimation, mapService,
                     case "feedback":
                         modalComment.blink();
                         break;
+                    case "movement-arrow":
```

Comment:
Love it, I'm glad that you were able to figure this out! One change I'd like to see is to add a function to do the blinking to `MapService.js` instead of having the functionality here; this way it matches all the other `blink()` functions.

The other thing I saw when looking at `MapService.js` is that we have functions `makeArrowsAndLinksClickable()` and `showNavigationArrows()` that are probably relevant to what you're doing. You might even be able to use the `status.svLinkArrowsLoaded` to know when you should try to find the arrows to make them blink!

Thoughts on that? I don't want you to have to spend a massive amount of time on this, especially since all of the tutorial code is hacky, and this is a one-off thing we're doing. My initial thought is that this will better fit the structure of our code. But you've also been looking at this longer, so let me know if this seems like a heavy lift! There's only so long that we should spend on making an arrow blink :joy: